frontend/src/pages/Setup/SetupRouter/SetupRouter.tsx

```diff
@@ -293,6 +293,24 @@ export const SetupRouter = () => {
 						<Text>Join community</Text>
 					</Link>
 				</Stack>
+
+				<Stack gap="0">
+					<Box mb="10" ml="8">
+						<Text
+							weight="bold"
+							size="xxSmall"
+							color="secondaryContentText"
+						>
+							Navigation
+						</Text>
+					</Box>
+					<NavLink
+						to="/new-correct-path"
+						className={styles.menuItemSecondary}
+					>
+						<Text>Back to Project</Text>
+					</NavLink>
+				</Stack>
```

Comment:
this adds a new button, this is in correct. there is a bug with the current button as described in the ticket